gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/main/java/io/gravitee/rest/api/service/impl/EnvironmentServiceImpl.java

```diff
@@ -18,8 +18,16 @@
 import io.gravitee.repository.exceptions.TechnicalException;
 import io.gravitee.repository.management.api.EnvironmentRepository;
 import io.gravitee.repository.management.model.Environment;
-import io.gravitee.rest.api.model.*;
-import io.gravitee.rest.api.service.*;
+import io.gravitee.rest.api.model.EnvironmentEntity;
+import io.gravitee.rest.api.model.MembershipEntity;
+import io.gravitee.rest.api.model.MembershipMemberType;
+import io.gravitee.rest.api.model.MembershipReferenceType;
+import io.gravitee.rest.api.model.UpdateEnvironmentEntity;
+import io.gravitee.rest.api.service.ApiHeaderService;
+import io.gravitee.rest.api.service.EnvironmentService;
+import io.gravitee.rest.api.service.MembershipService;
+import io.gravitee.rest.api.service.OrganizationService;
+import io.gravitee.rest.api.service.PageService;
```

Comment:
Can you check you're import settings because you're the only one with this kind of expended import 
<img width="1011" alt="image" src="https://user-images.githubusercontent.com/4112568/173510452-4759be6f-bf74-414e-9b3d-01a00c1d46bd.png">

Reply:
✅